cosigned/base/validatingwebhook.yaml

```diff
@@ -0,0 +1,20 @@
+---
+apiVersion: admissionregistration.k8s.io/v1
+kind: ValidatingWebhookConfiguration
```

Comment:
same as the `mutatingwebhookconfiguration` this is cluster resource, should go in:

https://github.com/operate-first/apps/tree/master/cluster-scope/base

Reply:
@cooktheryan you have moved it to cluster-scope.
however forget to remove it from here

Reply:
@harshad16 apologies just sent in patch